test/unit/mocks/entities.js

```diff
@@ -548,6 +548,7 @@ export const scheduledActionMock = {
   environment: { sys: cloneDeep(linkMock) },
   scheduledFor: {
     datetime: 'scheduledFor',
+    timezone: 'timezone',
```

Comment:
Should we make these somewhat plausible values? e.g.

```suggestion
    datetime: '2006-01-02T15:04:05-0700',
    timezone: 'Americas/Denver',
```